reference.md

```diff
@@ -664,33 +662,49 @@ Field Name | Presence | Type | Description
 
 ### wait_time
 
-This dynamic query provides wait time for specific location. To provide wait times to consumers, either this method or `wait_times.json` method can be used. `wait_time` allows more precise queries but requires a call on every interaction by users. 
+This dynamic query provides real time wait times for a specific location. To provide wait times to consumers, either this method or `wait_times.json` method can be used. `wait_time` allows more precise queries but requires a call on every interaction by users.
 
-The request must have the following query parameters. 
+The request must be a POST request (for security reasons) and the body use a JSON format. Here are the following attributes.
 
 Field Name | Presence | Type | Description
 ---|---|---|---
-`pickup_lat` | REQUIRED | Latitude | Latitude of the location where the user will be picked-up. 
-`pickup_lon` | REQUIRED | Longitude | Longitude of the location where the user will be picked-up. 
+`pickup_lat` | REQUIRED | Latitude | Latitude of the location where the user will be picked-up.
+`pickup_lon` | REQUIRED | Longitude | Longitude of the location where the user will be picked-up.
 `drop_off_lat` | Conditionally REQUIRED | Latitude | Latitude of the location where the user will be dropped off. Required if `drop_off_lon` is provided. FORBIDDEN otherwise.
 `drop_off_lon` | Conditionally REQUIRED | Longitude | Longitude of the location where the user will be dropped off. Required if `drop_off_lat` is provided. FORBIDDEN otherwise.
-`brand_id` | Conditionally REQUIRED | ID | Brand ID from `service_brands.json` to define the wait time is requested for which brand. REQUIRED if more than one service brand is available.  
+`brand_id` | REQUIRED | Array | Array of service brand Ids defined in `service_brands.json`. Used to filter out the results the wait time is requested for. At least one service brand Id must be provided.
```

Comment:
Curious as to the thought process here. Specifically, what is the intention behind enforcing at least one service brand ID?

Reply:
Changing from a single brand_id to an array of brand_id avoid making multiple chatty http calls. For example, it allows for requesting for ["taxi-registry-standard","taxi-registry-minivan","taxi-registry-special-need"] in one request. 

Enforcing at least on service brand ID, ask the person who consumes the api to specifies the brand_id that are required instead of asking for all brand_id by default.

Reply:
The switch to an array makes sense to me!